implementations/rust/ockam/ockam_api/src/cloud/enroll.rs

```diff
@@ -118,14 +121,15 @@ mod node {
             };
 
             trace!(target: TARGET, "authenticating token");
-            self.request_controller(
+            self.request_controller_with_timeout(
                 ctx,
                 api_service,
                 None,
                 cloud_route,
                 api_service,
                 req_builder,
                 ident,
+                Duration::from_secs(ORCHESTRATOR_RESTART_TIMEOUT),
             )
             .await
```

Comment:
My current understanding is that this request and the above request require the timeout.

However, I'm not entirely certain that Orchestrator projects are restarted in this process